contracts/src/deployers/HyperdriveDeployerCoordinator.sol

```diff
@@ -216,25 +231,23 @@ abstract contract HyperdriveDeployerCoordinator is
             target = IHyperdriveTargetDeployer(target0Deployer).deploy(
                 config_,
                 _extraData,
-                // NOTE: We hash the sender and deployment ID with the salt to
-                // prevent the front-running of deployments.
-                keccak256(abi.encode(msg.sender, _deploymentId, _salt))
+                // NOTE: We hash the deployment ID with the salt to prevent the
+                // front-running of deployments.
+                keccak256(abi.encode(_deploymentId, _salt))
             );
 
             // Store the deployment.
-            _deployments[msg.sender][_deploymentId].configHash = configHash_;
-            _deployments[msg.sender][_deploymentId]
-                .extraDataHash = extraDataHash;
-            _deployments[msg.sender][_deploymentId]
-                .initialSharePrice = initialSharePrice;
-            _deployments[msg.sender][_deploymentId].target0 = target;
+            _deployments[_deploymentId].configHash = configHash_;
+            _deployments[_deploymentId].extraDataHash = extraDataHash;
+            _deployments[_deploymentId].initialSharePrice = initialSharePrice;
+            _deployments[_deploymentId].target0 = target;
```

Comment:
does it make sense to use a storage pointer for _deployments[_deploymentId] in this function?